kickstart/PXELinux.erb

```diff
@@ -20,19 +20,40 @@ oses:
     options << "modprobe.blacklist=" + @host.params['blacklist'].gsub(' ', '')
   end
   options = options.join(' ')
+  # handle non-DHCP environments (e.g. bootdisk)
+  subnet = @host.subnet
+  if subnet.respond_to?(:dhcp_boot_mode?)
+    dhcp = subnet.dhcp_boot_mode? && !@static
+  else
+    dhcp = !@static
```

Comment:
`@static` should be removed, it's obsolete in favour of the subnet configuration